website/docs/d/pi_pvm_snapshots.html.markdown

```diff
@@ -0,0 +1,57 @@
+---
+
+subcategory: "Power Systems"
+layout: "ibm"
+page_title: "IBM: pi_pvm_snapshots"
+description: |-
+  Manages an pvm instance snapshots in the Power Virtual Server cloud.
+---
+
+# ibm_pi_pvm_snapshots
+Retrieve information about a Power Systems Virtual Server PVM instance snapshots. For more information, about Power Virtual Server PVM instance snapshots, see [getting started with IBM Power Systems Virtual Servers](https://cloud.ibm.com/docs/power-iaas?topic=power-iaas-getting-started).
+
+## Example Usage
+
+```terraform
+data "ibm_pi_pvm_snapshots" "ds_pvm_snapshots" {
+  pi_instance_name     = "terraform-test-instance"
+  pi_cloud_instance_id = "49fba6c9-23f8-40bc-9899-aca322ee7d5b"
+}
+```
+
+**Note**
+* Please find [supported Regions](https://cloud.ibm.com/apidocs/power-cloud#endpoint) for endpoints.
+* If a Power cloud instance is provisioned at `lon04`, The provider level attributes should be as follows:
+  * `region` - `lon`
+  * `zone` - `lon04`
+
+  Example usage:
+  
+  ```terraform
+    provider "ibm" {
+      region    =   "lon"
+      zone      =   "lon04"
+    }
+  ```
+
+## Argument reference
+Review the argument references that you can specify for your data source. 
+
+- `pi_instance_name` - (Required, String) The name of the instance.
+- `pi_cloud_instance_id` - (Required, String) The GUID of the service instance associated with an account.
+
+## Attribute reference
+In addition to all argument reference list, you can access the following attribute references after your data source is created. 
+
+- `pvm_snapshots` - List of PVM instance snapshots.
+  
+  Nested scheme for `pvm_snapshots`:
+  - `action` - (String) Action performed on the instance snapshot.
+  - `creation_date` - (String) Creation Date.
```

Comment:
Creation Date should be as The creation date.

Reply:
done